benchtable.js

```diff
@@ -160,7 +161,12 @@ BenchTable.prototype = Object.create(Benchmark.Suite.prototype);
 BenchTable.prototype.run = function(config) {
   var self = this, mapkey, mapfun;
 
-  function createBenchmarkFunction(fun, input) {
+  function createBenchmarkFunction(fun, options, input) {
```

Comment:
No real preference there. I chose to keep `fun` and `options` next to each other similarly to `addFunction`. This is an internal function only, so I have no problem switching it.